packages/webpack/test/fixtures/main/webpack.config.js

```diff
@@ -46,6 +46,9 @@ function makeConfig(lmOptions = {}) {
       }),
       new HtmlWebpackPlugin(),
     ],
+    resolve:{
+      fallback: { "crypto": false }
```

Comment:
guarantees `crypto` will be an ignored module instead of depending on default behavior that depends on other factors.

Reply:
wat

Reply:
I discovered the ignored modules by testing with an example that had `crypto` in it. Depending on the content of your node modules (and I didn't explore what else) it can get ignored or filled with a package. This setting is here to stabilize that and guarantee the test is still checking our support for ignored modules.